_resources/port1.0/group/github-1.0.tcl

```diff
@@ -26,7 +26,8 @@ options github.tarball_from
 default github.tarball_from tags
 option_proc github.tarball_from handle_tarball_from
 proc handle_tarball_from {option action args} {
-    global github.author github.project github.master_sites git.branch github.homepage
+    global github.author github.project github.master_sites git.branch github.homepage \
+           git.branch
```

Comment:
Sorry, I'm not a reviewer or anything, but I happened to look at the changes in this PR out of curiosity.

Is it intentional to have `git.branch` listed twice?

Reply:
Good catch - no, will fix.